backend/app/services/base.py

```
@@ -0,0 +1,49 @@
+from sqlmodel import Session
+
+from app.repositories.base import BaseRepository
+
+ModelType = TypeVar("ModelType")
+CreateSchemaType = TypeVar("CreateSchemaType")
+UpdateSchemaType = TypeVar("UpdateSchemaType")
+RepositoryType = TypeVar("RepositoryType", bound=BaseRepository)
+
+
+class BaseService(
+    Generic[ModelType, CreateSchemaType, UpdateSchemaType, RepositoryType]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)